guidelines/github-badges.md

```diff
@@ -0,0 +1,24 @@
+# Badges
+
+## Must have (order to be specified)
+
+- [license](https://shields.io/category/license) (it is sth people check first if they want to use repository)
+- [build status](https://help.github.com/en/articles/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository) (says if current code is working at all, depends on build service)
+- [last commit](https://shields.io/category/activity) (to inform when last update was done)
+- [dependencies](https://david-dm.org/) (to show that we care about our repos and update them as needed)
+- [contributors](./contributors.md) (will be added and updated automatically by "all-contributors" bot)
+- [code coverage](./code-coverage.md) (says about test existence and code quality)
+- [code quality](./code-quality.md)
+- [Security analysis](./security.md)
+
+[Any others?](https://shields.io/)
+
+## For NPM modules
+
+Should we duplicate github info that is seen above on page?
+
+Which page is more important Github or NPM?
```

Comment:
GitHub is a source of truth - NPM should be true treated as a package repository (immutable/read-only view of the repo).